common/src/java/com/zimbra/common/soap/MailConstants.java

```diff
@@ -18,6 +18,7 @@
 
 import org.dom4j.Namespace;
 import org.dom4j.QName;
+import org.python.antlr.ast.Str;
```

Comment:
is this import required

Reply:
No this is not required, I have removed it.